src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/ResponseExceptionHandler.java

```diff
@@ -47,7 +47,7 @@
     }
 
     @ExceptionHandler(InvalidCaseUpdateDetailsException.class)
-    protected ResponseEntity<ErrorResponse> handleInvalidCaseUpdateDetails(InvalidExceptionRecordException exc) {
+    protected ResponseEntity<ErrorResponse> handleInvalidCaseUpdateDetails(InvalidCaseUpdateDetailsException exc) {
```

Comment:
ha! and we all were blind. integration test saved us

Reply:
I removed `public` modifier in test classes. The other code smells are about using lambdas instead of try - catch in tests for exceptions (which causes worse readability).